proto/greptime/v1/ddl.proto

```diff
@@ -71,6 +71,7 @@ message CreateViewExpr {
   bytes logical_plan = 4;
   bool create_if_not_exists = 5;
   bool or_replace = 6;
+  repeated TableName table_names = 7;
```

Comment:
Why is this new field needed?

Reply:
Used to do validations. For example, we can validate if there are any views referenced to the table before the table is dropped. It's not used yet, but saved for future usage.